### ISSUE TITLE
Add testcases

### ISSUE THREAD

**Issue by strk** [larsop/content_balanced_grid#1]
Needed to show bugs :)

**Comment by larsop** [comment]
Yes, I assume that's not the first time from me hideing bugs :)

**Comment by strk** [comment]
Yeah, I'm after some bug with passing temporary views to the grid, I wanted to test it
